build.sbt

```diff
@@ -182,6 +185,28 @@ lazy val refined = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .nativeSettings(sharedNativeSettings)
   .dependsOn(core)
 
+lazy val iron = crossProject(JVMPlatform)
```

Comment:
To support JS and Native I guess you should add `JSPlatform` and `NativePlatform`.

You should also use `jsSettings` and `nativeSettings`.

Reply:
tx!, I'll look into JS and native once i have JVM working

Reply:
Decided to already add js/native as it does not seem to interfere with the problem i'm faced with